examples/db/database/main.tf

```diff
@@ -1,30 +1,6 @@
-resource "vkcs_db_instance" "db-instance" {
-  name        = "db-instance"
-
-  availability_zone = "GZ1"
-
-  datastore {
-    type    = "mysql"
-    version = "5.7"
-  }
-
-  flavor_id   = data.vkcs_compute_flavor.db.id
-
-  size        = 8
-  volume_type = "ceph-ssd"
-
-  network {
-    uuid = vkcs_networking_network.db.id
-  }
-
-  depends_on = [
-    vkcs_networking_router_interface.db
-  ]
-}
-
 resource "vkcs_db_database" "db-database" {
   name        = "testdb"
-  dbms_id     = vkcs_db_instance.db-instance.id
+  dbms_id     = vkcs_db_instance.basic.id
```

Comment:
lets name it basic_ms. otherwise charset/collate may confuse someone

Reply:
not basic - we use there capabilities and configgroup, so mysql

Reply:
yes i renamed to mysql and made all dependencies like mysql-db mysql-user etc